next.config.js

```diff
@@ -1,7 +1,9 @@
 const relayConfig = require('./relay.config');
 const { withSentryConfig } = require('@sentry/nextjs');
+const withRoutes = require('nextjs-routes/config')();
 
-const moduleExports = {
+/** @type {import('next').NextConfig} */
```

Comment:
This gives a nice autocomplete for next config options

Reply:
YESSSSSS